test/test-build.mjs

```diff
@@ -0,0 +1,5 @@
+import assert from 'assert';
+
+import { translations } from '@prezly/themes-intl-messages';
+
+assert(translations.actions.backToHomePage.id === 'actions.backToHomepage');
```

Comment:
Previously, the so-called ESM output was not compliant with the ESM module loader spec, and could only work post-processed by a bundler (which is more permissive to its ESM input). 

This test makes sure the output is actually compatible with the [Node.js ESM implementation](https://nodejs.org/api/esm.html), which is based on the [ES Modules spec](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules).

Reply:
I think it makes sense to outline the reason for this particular test setup in a short comment in both of the test files. I didn't get the reason behind it straight away, and having some context would help the future generations :)

Reply:
af1ae89ec0737684210644ff5e05f52bc8cf07a6